LocalSettings-proxy.txt

```diff
@@ -0,0 +1,5 @@
+// Allow proxying content from production
+
+$wgMFContentProviderClass = "MobileFrontend\\ContentProviders\\MwApiContentProvider";
+$wgMFContentProviderTryLocalContentFirst = false;
```

Comment:
This option should be dropped then, given what we say in the form below. Otherwise we would always get remote content.

Reply:
Switched to true